sites/partners/pages/sign-in.tsx

```diff
@@ -168,6 +169,8 @@ const SignIn = () => {
           setPhoneNumber,
           resetNetworkError
         )}
+        emailOnClick={() => setValue("mfaType", EnumRequestMfaCodeMfaType.email)}
```

Comment:
I'm curious why it's EnumRequestMfaCodeType.email instead of RequestType.email (and same for *.sms)?

Reply:
the `EnumRequestMfaCodeType` comes directly from our backend, which we can't have in ui-components, but we do want that type safety in partners. The `RequestType` is an intermediary solution to keep some type safety in that component but not depend on the actual backend. This whole component would need a bigger refactor in general to be less tied to Bloom - I imagine this prop needs to be something much more generic around MFA types a consumer could have.